<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMedDialog</class>
 <widget class="QWidget" name="AddMedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj lek</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLineEdit" name="medNameInput">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Nazwa leku</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="medDoseInput">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Dawka (np. 2x dziennie)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
